modules/table/src/test/java/org/apache/ignite/internal/table/TableManagerTest.java

```diff
@@ -600,16 +615,32 @@ private DataStorageManager createDataStorageManager(
             Path storagePath,
             RocksDbStorageEngineConfiguration config
     ) {
-        DataStorageManager manager = new DataStorageManager(registry, storagePath) {
-            /** {@inheritDoc} */
-            @Override
-            protected Iterable<StorageEngineFactory> engineFactories() {
-                return List.of((registry1, storagePath1) -> new RocksDbStorageEngine(config, storagePath1));
-            }
-        };
+        DataStorageManager manager = new DataStorageManager(
+                registry,
+                storagePath,
+                List.of(new StorageEngineFactory() {
```

Comment:
As it was last time, can we avoid anonymous class here?

Reply:
Fix it.